### PR TITLE
i18n(es): Update translation `tina-cms.mdx`

### PR DESCRIPTION
**Pull request by Waxer59**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
update the `tina-cms.mdx` translation

### CODE REVIEW

**Review by dreyfus92 — CHANGES_REQUESTED** [review]
Te dejo una sugerencia @Waxer59, buen trabajo 😁

File: src/content/docs/es/guides/cms/tina-cms.mdx [hunk 1]
```diff
@@ -143,3 +143,11 @@ Para empezar, necesitarás un proyecto Astro existente.
...
 
 5. En producción, TinaCMS puede hacer commit de los cambios directamente a tu repositorio de GitHub. Para configurar TinaCMS para producción, puedes elegir usar [Tina Cloud](https://tina.io/docs/tina-cloud/) o auto-hospedar la [Tina Data Layer](https://tina.io/docs/self-hosted/overview/). Puedes [leer más sobre el registro para Tina Cloud](https://app.tina.io/register) en la documentación de Tina.
+
+## Recursos Oficiales
+
+- [Guía de integración de TinaCMS con Astro](https://tina.io/docs/frameworks/astro/).
+
+## Recursos de la comunidad 
+
+- [Astro Tina Starter](https://github.com/tombennet/astro-tina-starter/tree/main) por Tom Bennet
```
Comment (dreyfus92) [inline src/content/docs/es/guides/cms/tina-cms.mdx:153]:
```suggestion
- [Plantilla de inicio de Tina con Astro](https://github.com/tombennet/astro-tina-starter/tree/main) por Tom Bennet
```

**Comment by Waxer59** [comment]
Gracias por la revisión 🤠

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks for this peeps 🚀

**Pull request merged**